api/controllers/site.js

```diff
@@ -27,7 +29,9 @@ const stripCredentials = ({ username, password }) => {
 
 module.exports = wrapHandlers({
   async findAllForUser(req, res) {
-    const user = await fetchModelById(req.user.id, User, { include: [Site] });
+    const user = await fetchModelById(req.user.id, User, {
```

Comment:
same as my comment on querying the organizations, but I know this was already here.

Reply:
Are you thinking of creating a scope that gets sites based on user and organization relationships? Or just use a scoped query based on user?

Reply:
just based on user